test/cpp/lazy/test_tensor_impl.cpp

```diff
@@ -6,12 +6,14 @@
 namespace torch {
 namespace lazy {
 
-// TODO(alanwaketan): Update the following unit tests once the TorchScript backend is merged.
+#ifdef FBCODE_CAFFE2
+// Lazy Tensor is disabled in FBCODE until addressing non-virtual methods (e.g. sizes) in TensorImpl
```

Comment:
If this ends up being important, the sizes thing is fixable - @swolchok recently landed something to fix a similar issue for NestedTensor: https://github.com/pytorch/pytorch/pull/73817